students/cbrown/Lesson 3/String Formatting.py

```diff
@@ -0,0 +1,43 @@
+#Lesson 3 String Formatting
```

Comment:
tip: avoid having spaces in python files names, every python file is a module by default which cannot have spaces